src/symbolize/gimli.rs

```diff
@@ -409,6 +426,43 @@ impl Cache {
     }
 }
 
+impl ShortBacktrace {
+    fn from_raw(raw: u8) -> Option<Self> {
+        let this = match raw {
+            0 => ShortBacktrace::ThisFrameOnly,
+            1 => ShortBacktrace::Start,
+            2 => ShortBacktrace::End,
+            _ => return None,
+        };
+        Some(this)
+    }
+}
+
+#[allow(non_upper_case_globals)]
+const DW_AT_short_backtrace: gimli::DwAt = gimli::DwAt(0x3c00);
```

Comment:
if this changes upstream it will also need to change here: https://github.com/llvm/llvm-project/pull/123683#discussion_r1923086452